explainability-core/src/main/java/org/kie/trustyai/metrics/drift/fouriermmd/Multiply.java

```diff
@@ -0,0 +1,30 @@
+package org.kie.trustyai.metrics.drift.fouriermmd;
+
+import java.util.function.Function;
+
+import org.kie.trustyai.explainability.model.Value;
+
+/*
+ * Accessory Function for transformColumn() calls.
+ */
+
+public class Multiply implements Function<Value, Value> {
+
+    private double multiplier;
+
+    public Multiply(double multiplier) {
+        this.multiplier = multiplier;
+    }
+
+    public Value apply(Value value) {
```

Comment:
This should check that the value.getType() are indeed Type.NUMBER first

Reply:
I drop the non-numeric columns in FourierMMD.getNumericColumns().  I left a (java) assert in Multiply, but it may not be necessary.  there is still a java assert in getNumericColumns() (asserting that there is at least one numeric column).